Monika After Story/game/script-moods.rpy

```diff
@@ -882,3 +882,26 @@ label mas_mood_excited:
             m 3eua "I know I'm always excited to see you every day."
             m 1hub "Either way, I'm glad that you're happy!"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_mood_database,
+            eventlabel="mas_mood_burnt",
+            prompt="...burnt out.",
+            category=[store.mas_moods.TYPE_BAD],
+            unlocked=True
+        ),
+        code="MOO"
+    )
+
+label mas_mood_burnt:
+    m 1wud "Oh!"
+    m 1ekd "I'm so sorry to hear you feel that way, [player]..."
+    m 2esc "But, if that's the case..."
+    extend "you really should consider taking a break, if you can."
```

Comment:
Is there any reason you're using `extend` here? It's used to perform an expression change mid-dialogue, but without one, there's no reason to use it.

Reply:
Ah...seems I forgot to add an expression change here.
I'll throw one in quick.